api/env.sample

```diff
@@ -19,4 +19,7 @@ OPENAI_KEY=
 AWS_ACCESS_KEY_ID=
 AWS_SECRET_ACCESS_KEY=
 AWS_IMGS_BUCKET=
-AWS_DEFAULT_REGION="us-west-2"
\ No newline at end of file
+AWS_DEFAULT_REGION="us-west-2"
+
```

Comment:
We are using US-East-1 for the mainline servers. Not sure if it matters for this. But one thing to be aware of is because of CI/CD when a new env var is added it has to be added in the GH Actions Yaml and in the Repo settings in GH